soh/soh/SaveManager.cpp

```diff
@@ -52,6 +53,10 @@ SaveManager::SaveManager() {
         info.randoSave = 0;
         info.requiresMasterQuest = 0;
         info.requiresOriginal = 0;
+
+        for (int i = 0; i < ARRAY_COUNT(info.buildVersion); i++) {
+            info.buildVersion[i] = '\0';
+        }
```

Comment:
does `memset` not work here?
```suggestion
       memset(&info.buildVersion[0], 0, sizeof(info.buildVersion));
```